src/inmanta/ast/type.py

```diff
@@ -762,7 +765,7 @@ def resolve_type(locatable_type: LocatableString, resolver: Namespace) -> Type:
     # quickfix issue #1774
     allowed_element_type: Type = Type()
     if locatable_type.value == "list":
-        return TypedList(allowed_element_type)
+        return List()
     if locatable_type.value == "dict":
```

Comment:
Looking at this code I expect the same issue for dict. Could you check that as well?

Reply:
It seems we dont have the issue with dicts due to the usage of `type_string_internal` and `type_string` in TypedDict and Dict respectively

Reply:
I've added a test